haswell/haswell_memory_bound.xml

```diff
@@ -109,22 +175,29 @@
             divideBySampleTime="false" />
         <display>
             <displayName>Stalled for other reasons</displayName>
-            <description>Fraction of cycles that are stalled for non-memory related reasons</description>
+            <description>Fraction of stalled cycles that are stalled for non-memory related reasons</description>
             <type>other</type>
             <colour>SpecialLine6</colour>
         </display>
     </metric>
+-->
 
     <metricGroup id="Haswell_papi_memory_boundedness">
         <displayName>MemoryBound</displayName>
         <description>Gives a measure of how memory bound an application is. This is only accurate on Intel Haswell (Xeon v3) cores</description>
         <metric ref="haswell.papi.active_cycles"/>
         <metric ref="haswell.papi.productive_cycles"/>
-        <metric ref="haswell.papi.stall_cycles" />
+        <metric ref="haswell.papi.stall_cycles"/>
+	<metric ref="haswell.papi.store_buffer_stall_cycles"/>
+	<metric ref="haswell.papi.l1d_pending_stall_cycles"/>
         <metric ref="haswell.papi.memory_bound" />
+	<metric ref="haswell.papi.l1d_pend_miss_fb_full_cycles"/>
+	<metric ref="haswell.papi.offcore_requests_buffer_sq_cycles"/>
         <metric ref="haswell.papi.bandwidth_bound" />
+<!--
```

Comment:
Why are these commented out? If you don't need them, just remove them.

Reply:
I will remove them.